.github/workflows/validate-this-gha-refs.yml

```diff
@@ -0,0 +1,10 @@
+name: Validate GHA Refs
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  validate-gha-references:
+    name: Validate GHA Refs
+    uses: Lombiq/GitHub-Actions/.github/workflows/validate-gha-refs.yml@main
```

Comment:
Good catch, thank you. For now, I had to change to `issue/OSOE-517` so that the [validate-gha-refs](https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-517/.github/workflows/validate-this-gha-refs.yml) workflow could run on the push to the `issue/OSOE-517` branch.

Reply:
Not sure if you'll need that, but just wanted to mention that you can grab the default of the current repo with `${{ github.event.repository.default_branch }}`.

Reply:
Thanks @BenedekFarkas - Yes, I can use that instead of hardcoding `dev`